### PR TITLE
Portfolio Project mobile menu

### PR DESCRIPTION
**Pull request by Profsain**
Added mobile menu

### CODE REVIEW

File: asset/script/main.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+function closeNav() {
+  hamburgar.classList.remove('change');
+  mobileMenu.classList.remove('open');
+}
+
+hamburgar.addEventListener('click', toggleMobileMenu);
+
+navClose.forEach((elem) => {
+  elem.addEventListener('click', closeNav);
+});
```
Comment (KayLemba) [inline asset/script/main.js]: While you did a good job Implementing this function unfortunately your hambuger button isnt visible. Please make sure the hamburger is visible when user switches to the mobile view.

Look at your design:
Screenshot from 2022-05-10 15-37-31

Your app should look like this: 
Screenshot from 2022-05-10 15-47-05

And then after the hamburger is clicked it should display this:
Screenshot from 2022-05-10 15-47-19

**Review by KayLemba — CHANGES_REQUESTED** [review]
Hi @Profsain  👋
Good job so far working on the project requirements!
There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight 🎉
No linter errors. ✔️
GitHub flow is followed. ✔️
Professional README. ✔️

Required Changes ♻️
Check the comments under the review in the inline comments.

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: asset/script/main.js [hunk 2]
```diff
@@ -1,15 +1,18 @@
 const mobileMenu = document.querySelector('.mobile-menu');
 const hamburgar = document.getElementById('hamburger-icon');
 const navClose = document.querySelectorAll('.mobile-nav-link');
+const brand = document.querySelector('#brand');
 
 function toggleMobileMenu() {
   mobileMenu.classList.toggle('open');
   hamburgar.classList.toggle('change');
+  brand.classList.toggle('closeBrand');
 }
```
Comment (chrystalme) [inline asset/script/main.js:10]: Use ES6 syntax for your funtions by using arrow function

File: asset/script/main.js [hunk 3]
```diff
@@ -1,15 +1,18 @@
...
 
 function closeNav() {
   hamburgar.classList.remove('change');
   mobileMenu.classList.remove('open');
+  brand.classList.remove('closeBrand');
 }
```
Comment (chrystalme) [inline asset/script/main.js:16]: Use ES6 syntax for your funtions by using arrow function. eg `const closeNav = () => {
.
.
.
}`

**Review by chrystalme — APPROVED** [review]
## Hi @Profsain and @henrycode460,

## Project Status ✅ Approve
image


In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 

## Optional Changes

- [ ] Check the inline comments. Kindly use ES6 arrow functions.
- [ ] You could replace all the HTML tags in CSS default overwrite to `*`

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Also, you can reach me on [Twitter](https://twitter.com/afam_ifeanyi) or [Slack](https://microverse-community.slack.com/team/U019ZC99KFA)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**